blenderAddons/vispCAOExport/setup.py

```diff
@@ -0,0 +1,9 @@
+import bpy
+import sys
+import os
+
+print("BLENDER VERSION : " + bpy.app.version_string)
+ADDON_PATH = (bpy.utils.user_resource('SCRIPTS', "addons"))
+
+os.system("cp -r ../vispCAOExport " + ADDON_PATH)
+print("Moved addon to " + ADDON_PATH)
```

Comment:
Some comments:
- A new line should be introduced at the end of the file ;-)
- This script is not cross-platform since `cp -r` doesn't exists on Windows. There is `python shutil` that may help (see below)
- The script is not working os OSX since `ADDON_PATH=/Users/fspindle/Library/Application Support/Blender/2.78/scripts/addons` folder doesn't exists. What I have is the following:
````
$ ls /Users/fspindle/Library/Application\ Support/Blender/2.78/
config
````
- the script should be improved to created the destination folder recursively. A good candidate is to use: `shutil.copytree("../vispCAOExport", ADDON_PATH)`
- the message should be `Copied addon vispCAOExport to `
- python exceptions should be catched to warn the user if the copy failed